Yoast/YoastSEO.js
# Add adjective morphology

## Summary

This PR can be summarized in the following changelog entry:

* Implement morphological analysis for English adjectives

## Test instructions

This PR can be tested by following these steps:

* Try adding more word arrays to the spec, and see if those pass in yarn test
* Run grunt build:js. Then in browerfied, enter a keyword and see if all forms (base, comparative, superlative, adverb) are returned
* Check if the keyword density is correct

Fixes #1557
